### DISCUSSION TITLE
How to access Flink UI?

### DISCUSSION THREAD

**Discussion by cutiechi** [apache/kyuubi#2141] [Q&A] [answered]
How to access flink ui?

**Comment by yaooqinn** [comment]
cc @yanghua @SteNicholas

**Comment by a49a** [comment]
You can read this doc https://nightlies.apache.org/flink/flink-docs-release-1.14/docs/deployment/resource-providers/standalone/overview/

bin/start-cluster.sh and access http://localhost:8081

**Comment by pan3793** [comment]
To be clear, in current design, if you are running `kyuubi-flink-sql-engine` in one of "local cluster"/"yarn session" mode, Kyuubi has no responsibility for Flink Cluster lifecycle management, that means, you should prepare a Flink Cluster before launching Flink Jobs in Kyuubi.

**Reply by cutiechi** [reply]
but when i execute sql there is no job on flink ui

**Reply by cutiechi** [reply]
@pan3793

**Comment by a49a** [comment, marked as answer]
You can read this doc https://kyuubi.readthedocs.io/en/latest/quick_start/quick_start.html#execute-flink-sql-statements

After you run `INSERT` statement, I will check out a job in the `running jobs` or `completed jobs` tabs.

**Reply by cutiechi** [reply]
thx
